packages/lesswrong/components/ea-forum/EAHome.tsx

```diff
@@ -31,8 +31,11 @@ const EAHome = () => {
       {shouldRenderEventBanner && <EventBanner />}
       
       <StickiedPosts />
-      
-      {reviewIsActive() && <FrontpageReviewWidget />}
+
+      {/* EA forum look here */}
```

Comment:
the reason for this is that on the ReviewVotingPage, I wanted to use the FrontpageReviewWidget, but the size of the column didn't precisely correspond to the SingleColumnSeciton that FrontpageReviewWidget normally wrapped itself in.

Reply:
SingleColumnSection includes a margin-bottom. I think it is plausible we should be replacing the architecture with a... _single_ singleColumnSection + dividers in between. But probably can wait till this comes up a second time.